modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists on your heterogeneous cluster 
+
+On a {product-title} {product-version} heterogeneous cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option in order to import the manifest list.
+
+.Prerequisites 
+
+* You installed the Openshift CLI (`oc`).
+
+.Procedure 
+
+* You can change the default `importMode` option by running the following command: 
+
++
+[source,terminal]
+----
+oc patch is/cli-artifacts -n openshift -p '{"spec":{"tags":[{"name":"latest","importPolicy":{"importMode":"PreserveOriginal"}}]}}'
+----
```

Comment:
One more important note we need to add is that this will only work for imagestreams which have the referencePolicy as source in their spec.

```
 referencePolicy:
      type: Source
```

Reply:
or the alternative is to set the policy to source if that imagestream is really needed to work on the heterogeneous cluster across the different arch nodes

Reply:
the reference type will be present in every imagestream, it will be either local or source and yes it can be found in the imagestream yaml